modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/js/DLFolderDropdownPropsTransformer.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openConfirmModal, openModal} from 'frontend-js-web';
+
+const _webDAVHTML = (
+	learnMessage,
+	learnURL,
+	portletNamespace,
+	webDavURL
+) => `<div class="portlet-document-library">
+		${Liferay.Language.get('webdav-help')}
+
+		<a href="${learnURL}" target="_blank">${learnMessage}</a>
+		
+		<br/><br/>
+		
+		<div class="form-group input-resource-wrapper">
+			<label class="control-label" for="${portletNamespace}webDavURL">
+				${Liferay.Language.get('web-dav-url')}
+			</label>
+
+			<input class="form-control lfr-input-resource" disabled id="${portletNamespace}webDavURL" name="${portletNamespace}webDavURL" type="text" value="${webDavURL}"/>
+		</div>
+	</div>`;
+
+const ACTIONS = {
+	accessFromDesktop({learnMessage, learnURL, webDavURL}, portletNamespace) {
+		openModal({
+			bodyHTML: _webDAVHTML(
+				learnMessage,
+				learnURL,
+				portletNamespace,
+				webDavURL
+			),
+			onOpen() {
+				const webdavURLInput = document.getElementById(
+					`${portletNamespace}webDavURL`
+				);
+
+				if (webdavURLInput) {
+					webdavURLInput.focus();
+				}
+			},
+			title: Liferay.Language.get('access-from-desktop'),
+		});
+	},
+
+	delete({deleteURL}) {
+		openConfirmModal({
+			message: Liferay.Language.get(
+				'are-you-sure-you-want-to-delete-this'
+			),
+			onConfirm: (isConfirmed) =>
+				isConfirmed && submitForm(document.hrefFm, deleteURL),
+		});
+	},
+
+	deleteExpiredTemporaryFileEntries({deleteExpiredTemporaryFileEntriesURL}) {
+		submitForm(document.hrefFm, deleteExpiredTemporaryFileEntriesURL);
```

Comment:
Hey @ealonso, this behaviour is different now... Before we just had 
```
<liferay-ui:icon
	message="delete-expired-temporary-files"
	url="<%= folderActionDisplayContext.getDeleteExpiredTemporaryFileEntriesURL() %>"
/>
```

It was a direct navigation to that url, but now we are submitting the form... is this intented? does it work? I don't know how to test this, could you pass me steps to reproduce it?